src/yadism/coefficient_functions/fonll/g1_nc_raw.py

```diff
@@ -388,3 +392,50 @@ def c2g_NNLL_reg(z, _args):
         )
     ) / (9.0)
     return result.real
+
+
+@nb.njit("f8(f8,f8[:])", cache=True)
+def pdf_matching_NNLL_loc(z, args):
+    l = np.log(z)
+    l2 = l**2
+    lm = np.log(1 - z)
+    lm2 = lm**2
+    Li2 = li2(z)
+    Li2m1 = li2(1 / (1 - z))
+    Li3 = nielsen(2, 1, z)
+    aqq_loc = (1 / 54) * (
+        3 * (73 + 6 * np.pi**2)
+        + 8
+        * (
+            -5 * np.pi**2
+            + 56 * lm
+            - 30 * 1j * np.pi * lm
+            + 3 * np.pi**2 * lm
+            + 15 * lm2
+            + 9 * lm * l2
+            + 30 * Li2m1
+            + 18 * l * Li2
+            - 18 * Li3
+        )
+    ).real
+    return TR * CF * aqq_loc + 1 / 4 * beta0Q * 2 * lo.pqq_local(z, args) * zeta2
+
+
+@nb.njit("f8(f8,f8[:])", cache=True)
+def pdf_matching_NNLL_reg(z, args):
+    aqq_reg = -(2 / 27) * (
+        -22
+        + 134 * z
+        + 3 * np.pi**2 * (1 + z)
+        + (-6 + 66 * z) * np.log(z)
+        + 9 * (1 + z) * np.log(z) ** 2
+    )
+    return TR * CF * aqq_reg + 1 / 4 * beta0Q * 2 * lo.pqq_reg(z, args) * zeta2
+
+
+@nb.njit("f8(f8,f8[:])", cache=True)
+def pdf_matching_NNLL_sing(z, args):
+    aqq_sing = (
+        -224 / 27 - 4 / 9 * np.pi**2 - 40 * np.log(z) / 9 - 4 * np.log(z) ** 2 / 3
+    ) / (z - 1)
+    return TR * CF * aqq_sing + 1 / 4 * beta0Q * 2 * lo.pqq_sing(z, args) * zeta2
```

Comment:
I think the convention of adding the factor of 2 (both here and in eko) in front of each splitting function coming from https://arxiv.org/pdf/2211.15337.pdf make sense.
As you can see from the tests the LL and NLL pieces are the same as in the unpolarised case and they do match the eko implementation. 
So I'd say from this point of view we are cool with all the conventions!

NOTE: for completeness I believe that in https://arxiv.org/pdf/2211.15337.pdf, splitting functions are following the convention of https://arxiv.org/pdf/hep-ph/9508347.pdf where the DGLAP solution (2.10) contains an addition factor of 1/2 in the exponent.

Reply:
Great! So this then solves the mystery we talked about yesterday.